contracts/token-manager/TokenManager.sol

```diff
@@ -100,23 +100,19 @@ abstract contract TokenManager is ITokenManager, ITokenManagerType, Operatable,
      */
     function setup(bytes calldata params) external override(Implementation, IImplementation) onlyProxy {
         bytes memory operatorBytes = abi.decode(params, (bytes));
-        address operator;
-
-        /**
-         * @dev Specifying an empty operator will default to the service being the operator. This makes it easy to deploy
-         * remote interchain tokens without knowing anything about the service address at the destination.
-         */
-        if (operatorBytes.length == 0) {
-            operator = address(interchainTokenService);
-        } else {
-            operator = operatorBytes.toAddress();
 
-            // Add flow limiter role to the service by default. The operator can remove this if they so choose.
-            _addAccountRoles(address(interchainTokenService), 1 << uint8(Roles.FLOW_LIMITER));
+        address operator = address(0);
```

Comment:
assigning to address(0) wastes gas

Reply:
slither warning. we do this elsewhere too so i think it's fine as a good practice